Docs/source/developers/testing.rst

```diff
@@ -81,6 +81,18 @@ For easier debugging, it can be convenient to run the tests on your local machin
 
        ctest --test-dir build -E laser_acceleration
 
+* Sometimes two or more tests share a large number of input parameters and differ by a small set of options.
+  Such tests typically also share a base string in their names.
+  For example, you can find three different tests named ``test_3d_langmuir_multi``, ``test_3d_langmuir_multi_nodal`` and ``test_3d_langmuir_multi_picmi``.
+  In such a case, if you wish to run the test ``test_3d_langmuir_multi`` only, this can be done again with the ``-R`` regular `expression filter <https://regex101.com>`__ via
+
+  .. code-block:: sh
+
+       ctest --test-dir build -R "test_3d_langmuir_multi\..*"
+
+  Note that filtering with ``-R "test_3d_langmuir_multi"`` would include the additional tests that have the same substring in their name and can be sufficient if one aims to isolate a single test.
```

Comment:
That's what I meant:
```suggestion
  Note that filtering with ``-R "test_3d_langmuir_multi"`` would include the additional tests that have the same substring in their name and would not be sufficient to isolate a single test.
```

Reply:
Ok, but I used could because this is just an example - not all examples are 5x oversubscribed...

Reply:
Ah I see. Here I was using the negative form because I was referring to the specific example in the text, of the three Langmuir tests with shared substring. So I meant to connect this sentence with that example, since it's in the same list item. That's why I got confused.